mkt/regions/migrations/0001_initial.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+
+from mkt.constants import regions
+from mkt.developers.cron import exclude_new_region
+
+
+def exclude_romania(apps, schema_editor):
+    exclude_new_region([regions.RO])
+
+
+class Migration(migrations.Migration):
+    operations = [
+        migrations.RunPython(exclude_romania),
```

Comment:
Why ? It's not a new region, it was already there. Also, the constant you are looking for is `ROU`, not `RO`